src/reducers/TasksReducer.js

```diff
@@ -1,20 +1,37 @@
+const _ = require('lodash')
+
 function updateTasks(existingTasks, { tasks, filter }) {
   const tasksByKey = {}
 
   // Add the existing existingTasks
   existingTasks.forEach(task => (tasksByKey[task.storageKey] = task))
 
   // Update existingTasks with new values and add new existingTasks
-  tasks.forEach((newTask) => {
-    const oldTask = tasksByKey[newTask.storageKey] || {}
-    const filterQueries = oldTask.filterQueries || []
-    if (!filterQueries.includes(filter.query)) {
-      filterQueries.push(filter.query)
+  tasks.forEach((updatedTask) => {
```

Comment:
You mentioned it yesterday, but I personally like the more terse style of not having the parentheses around a single parameter, and only adding them when we have to when we have multiple parameters.